src/runtime_src/core/common/sysinfo.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+#ifndef _xrt_core_common_sysinfo_h_
+#define _xrt_core_common_sysinfo_h_
+
+// Local - Include Files
+#include "config.h"
+#include "core/common/detail/sysinfo.h"
```

Comment:
This header file should be included in .cpp.   That will also fix the pipeline error.

Reply:
I need to include detail/sysinfo.h in the header file since utils.cpp calls "xrt_core::sysinfo::get_os_info(pt_os_info)" which is defined in detail/sysinfo.h

Reply:
No, you cannot do this.    Your core/common/sysinfo.h must provide APIs needed by your clients.